src/main/java/com/blamejared/crafttweaker/impl/brackets/BracketHandlers.java

```diff
@@ -179,6 +183,21 @@ public static BlockState getBlockState(String name, String properties) {
         return axis;
     }
     
+    /**
+     * Gets the equipment slot type based on name. Throws an error if it can't find the equipment slot type.
+     * @param tokens The equipment slot type's name
+     * @return The found equipment slot type
+     * @docParam tokens "mainhand"
+     */
+    @ZenCodeType.Method
+    @BracketResolver("equipmentSlotType")
```

Comment:
Ideally we want all BracketHandlers to be lowercase, so `damagesource` and `equipmentslottype`

Reply:
https://github.com/CraftTweaker/CraftTweaker/blob/fd0fcad4da4a943812101164d86d11f365eacb73/src/main/java/com/blamejared/crafttweaker/impl/brackets/BracketDumpers.java#L39-L43

But BracketDumpers are lower camel case.

Reply:
It does not matter, the prefixed-BEP will make it lowercased regardless iirc

Reply:
I replied in wrong place. I had made Bracket Handlers lower case. But I still leave Bracket Dumpers lower camel case.